nomad/structs/csi.go

```diff
@@ -350,50 +374,65 @@ func (v *CSIVolume) Copy() *CSIVolume {
 		out.WriteAllocs[k] = v
 	}
 
+	for k, v := range v.ReadClaims {
```

Comment:
do we need to copy the claims themselves here too? the `state` is mutable

Reply:
Good catch! Will fix!

Reply:
Done